_episodes/12-cluster.md

```diff
@@ -46,6 +47,13 @@ tasks.
 
 
 ## Logging in
+The first step in using a cluster is to establish a connection from our laptop to the cluster,
+via the internet. We use a program called the Secure SHell or ssh for this. (make sure you have 
```

Comment:
But for many institution-specific HPC systems access will be via the campus network rather than the Internet (at least when there's not a pandemic going on!).

How about "via the Internet and/or your organisation's network"?

Reply:
a544bb2